vassal-app/src/main/resources/VASSAL/i18n/VASSAL_it.properties

```diff
@@ -601,8 +601,8 @@ PlayerRoster.become_observer=Diventa osservatore
 PlayerRoster.join_another_side=Cambia fazione
 PlayerRoster.give_up_position=Abbandoni la fazione di %1$s?
 PlayerRoster.join_game_as=Scelta fazione  
-PlayerRoster.switch_sides=Il tuo lato attuale � %1$s. Passare da che parte?
-PlayerRoster.switch_sides2=%1$s ora non � disponibile. Il tuo lato attuale � %2$s. Passare da che parte?
+PlayerRoster.switch_sides=Il tuo lato attuale e' %1$s. Passare da che parte?
+PlayerRoster.switch_sides2=%1$s ora non e' disponibile. Il tuo lato attuale e' %2$s. Passare da che parte?
```

Comment:
That doesn't look right either? Is it supposed to be `é`?

IIRC, your options here are UTF-8 or `\u` escapes.

Reply:
Thanks. I will review again using UTF-8 and escapes. I noticed that the polish translation was using escapes. It seemed to me that most of the others were not (using UTF-8 perhaps?).

Also, some of the translation files now contain extra indecipherable (to me) characters. I'd noticed these before in the jp / zh translations but not in es or fr. For example, the middle line here:

<img width="447" alt="image" src="https://github.com/vassalengine/vassal/assets/58135975/6909da23-c9b8-4938-b08e-f7ca3ff064b0">